README.md

```diff
@@ -24,7 +24,7 @@
 
 ## Bootstrap 4
 
-Our default branch is for development of our upcoming Bootstrap 5 release. Head to the [`v4-dev` branch](https://github.com/twbs/bootstrap/tree/v4-dev) to view the readme, documentation, and source code for Bootstrap 4.
```

Comment:
Both statements (original and new) are grammatically correct. This change is superfluous.